ci/Jenkinsfile.desktopbuild

```diff
@@ -87,160 +95,163 @@ def buildClojureScript() {
   """
 }
 
-parallel(
-  'MacOS build': {
-    timeout(90) {
-      node('macos1') {
-        load "$HOME/env.groovy"
-
-        try {
-          stage('Git & Deps') {
-            slackNotify('MacOS build started.')
-            checkout scm
-            doGitRebase()
-            cleanupAndDeps()
-          }
+node() {
+  gitHubNotify()
```

Comment:
parallel build job was wrapped into `node` block to call `gitHubNotify()` once on the start of build job process